src/rime/dict/table.h

```diff
@@ -40,11 +40,7 @@ using Syllabary = Array<StringType>;
 
 using Code = List<SyllableId>;
 
-#if defined(__arm__)
 using Weight = double;
-#else
-using Weight = float;
-#endif
```

Comment:
統一用double的了？那bin會比較大。

Reply:
arm 原來就用 `double`。x86 的機器對文件大小不敏感吧。

Reply:
那倒是。不過我本地編的apk是都換成float的，小一點。

Reply:
I recalled that I changed to double because (in initial implementation) unaligned float caused SIGBUS error on ARM.
On x86 unaligned float only causes a slow down.
In this PR data are aligned for all architectures so we can safely change back to float.

Reply:
原來如此，那就放心地用float了。